src/gas-api.js

```diff
@@ -4,215 +4,229 @@ class Gas {
-        return hash;
-      }, {});
-      return responses;
-    })
-  }
-
-  gasCreateFile(file, type) {
-    const typeId = type === context.config.filetype ? 0 : 2;
-    const payload = `7|1|7|${getBaseUrl()}\|${context.gasToken}|_|makeNewFile|1a|i|${file}|1|2|3|4|2|5|6|7|6|${typeId}|`;
-    return new Promise((resolve, reject) => {
-      $.ajax({
-        url: context.gasUrl,
-        headers: context.gasHeaders,
-        method: 'POST',
-        crossDomain: true,
-        data: payload,
-        dataType: 'text'
       })
-      .then(resolve)
-      .fail((err) => {reject(new Error('Create file failed'))})
-    })
-    .then((response) => {
-      if (!response.startsWith('//OK')) throw(new Error(`Create file '${file}.${type}' failed`));
-      const responseData = eval(response.slice(4)).filter((e) => {
-        return typeof(e) === 'object';
-      })[0];
-      const id = responseData.filter((data) => { return /^\w{8}-\w{4}-\w{4}-\w{4}-\w{12}$/.test(data) });
-      if (id.length > 0) {
-        context.fileIds[file] = id[0];
-        return id[0];
-      } else {
-        throw new Error('can not parse response');
-      }
-    });
+      .then((response) => {
+        if (!response.startsWith('//OK')) throw (new Error(`Create file '${file}.${type}' failed`));
+        const responseData = eval(response.slice(4)).filter((e) => {
+          return typeof (e) === 'object';
+        })[0];
+        const id = responseData.filter((data) => {
+          return /^\w{8}-\w{4}-\w{4}-\w{4}-\w{12}$/.test(data)
+        });
+        if (id.length > 0) {
+          context.fileIds[file] = id[0];
+          return id[0];
+        } else {
+          throw new Error('can not parse response');
+        }
+      });
   }
 
   gasUpdateFile(file, code) {
     const escapedCode = code.replace(/\\/g, '\\\\').replace(/\|/g, '\\!');
     const payload = `7|1|7|${getBaseUrl()}\|${context.gasToken}|_|updateFile|1a|${file}|${escapedCode}|1|2|3|4|2|5|5|6|7|`;
     let headers = context.gasHeaders;
-    Object.assign(headers, { 'file-id': context.fileIds[file]});
+    Object.assign(headers, {
+      'file-id': context.fileIds[file]
+    });
     return new Promise((resolve, reject) => {
       $.ajax({
-        url: context.gasUrl,
-        headers: headers,
-        method: 'POST',
-        crossDomain: true,
-        data: payload,
-        dataType: 'text'
-      })
-      .then((response) => {
-        if (!response.startsWith('//OK')) reject(new Error('Update file failed'));
-        resolve();
-      })
-      .fail((err) => {
-        reject(new Error('Update file failed'));
-      });
+          url: context.gasUrl,
+          headers: headers,
+          method: 'POST',
+          crossDomain: true,
+          data: payload,
+          dataType: 'text'
+        })
+        .then((response) => {
+          if (!response.startsWith('//OK')) reject(new Error('Update file failed'));
+          resolve();
+        })
+        .fail((err) => {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/15f4157afeba0560f3f12968d583a6f4fceda1cf/src/gas-api.js#L202"></span>‘err’ is defined but never used.</td>
    </tr>
  </tbody>
</table>